fitnesse/resources/bootstrap-plus/templates/skeleton.vm

```diff
@@ -47,13 +47,13 @@
 
   <link rel="stylesheet" type="text/css" href="${contextRoot}files/fitnesse/wysiwyg/wysiwyg.css" media="screen"/>
   <link rel="stylesheet" type="text/css" href="${contextRoot}files/fitnesse/bootstrap-plus/css/font-awesome.min.css">
-
+  <link rel="csv file" type="text/csv" href="${contextRoot}files/fitnesse/bootstrap-plus/csv/toolTipData.csv">
```

Comment:
Wat doet deze link? Is de csv niet benaderbaar als 'ie niet gelinkt is oid?

Reply:
ik dacht idd dat ie toen idd niet benaderbaar was destijds, ik zie nu dat i dat wel is zonder.